tests/testthat/test-CreatePackageReport.R

```diff
@@ -106,6 +105,24 @@ test_that("CreatePackageReport rejects bad report_path arguments", {
 })
 
 
+test_that("CreatePackageReport respects report_path when explicitly given", {
+
+    testing_file <- tempfile(pattern = "output", fileext = ".html")
```

Comment:
I am covering that case with the `CreatePackageReport rejects bad paths` tests.

https://github.com/jameslamb/pkgnet/blob/path_test/tests/testthat/test-CreatePackageReport.R#L83

Reply:
@jameslamb What I meant was, if I specify a report_path of something like `'~/Desktop/myreport.html'`, then it will get respected and work as expected.

I think you can test this by doing the same test but putting the report in a temp directory.

Reply:
That is already being tested in this test! `tempfile()` generates a full path, not just a `file.extension`:

```
> tempfile(pattern = "output", fileext = ".html")
[1] "/var/folders/42/xp1wzfw555z9lkgb_n8qvwt8pm028j/T//RtmpzyE0Ja/output16a73df80e6d.html"
```